crepecake/src/main/java/com/google/cloud/tools/crepecake/registry/BlobPuller.java

```diff
@@ -54,6 +56,11 @@ public BlobHttpContent getContent() {
     return null;
   }
 
+  @Override
+  public List<String> getAccept() {
+    return Collections.emptyList();
```

Comment:
Blob Puller accepts nothing? or everything?

Reply:
Blobs are only sent as application/octet-streams, so the Accept header is optional/doesn't do anything here.